DebreGolgotaDibqGasha/DGDG001

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="NazMar001" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>The 'Sane Golgota'  collection of  prayer book</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="GA"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="ETH0001DibqGasha"/>
```

Comment:
Dear @Getinet309 ,

thank you for the collaboration! I think there is a some confusion with the ids for the repository and works which you have used in the entry. I could not find the ID ETH0001DibqGasha in our database, and judging from the prefix (ETH) it should refer to an ethnic group, but not to an institution/repository.

Reply:
Dear Daria,
Thank you I will change it, but the ms is not in the Betametsahft repository. A new ID is required.

Reply:
Dear Getinet,

sure, and you have created a new ID for the manuscript, which is correct! Now, you might search in the database if we already have an entry for the monastery, in which the manuscript is kept. I have conducted a preliminary search, but I could not find it. You could cross-check again, and in case we do not have an entry for this monastery, you could create a new record for it as well. (https://betamasaheft.eu/Guidelines/?id=institution)